terracumber/terraformer.py

```diff
@@ -43,9 +43,13 @@ def __init__(self, terraform_path, maintf, backend, variables={}, output_file=Fa
                     '%s/modules/backend' % terraform_path)
 
     def inject_repos(self, custom_repositories_json):
-        """Set additional repositories into the main.tf, so they are injected by sumaform"""
+        """Set additional repositories into the main.tf, so they are injected by sumaform
+
+        Returns True in case of success and False in case of error
```

Comment:
What error? It's a very specific error.

Reply:
Specify what the error is, so reading the code is not needed :-)

Reply:
@Bischoff if it evolves, then you will need to change this to numeric values, to specify the different errors, so for now document the current situation.

Reply:
Or just change it already to numeric values, and document the current list, as you prefer ;-)

Reply:
YAGNI, and also I now commented at the current check below.